src/hotspot/cpu/aarch64/aarch64_vector_ad.m4

```diff
@@ -2320,6 +2320,69 @@ instruct reinterpret_resize_gt128b(vReg dst, vReg src, pReg ptmp, rFlagsReg cr)
   ins_pipe(pipe_slow);
 %}
 
```

Comment:
The following hunk does not seem to be making good use of the macro processor.

Reply:
I updated m4 file. Please help to review, thanks!